src/templates/Footer/__tests__/footer.spec.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { expect } from 'chai';
+import { shallow } from 'enzyme';
+
+import Footer from 'templates/Footer';
+
+/* eslint-disable */
+jest.mock('atoms/Icon', () => {
```

Comment:
So, the only way I can tell to do this AND maintain the automated behavior is with the script I spoke about that creates the loader inside of the `require`.. it's much less elegant, and this is actually what the mocks are specifically used for. We could extract it into a neat helper method, though, that will mock any components we tell it to.